### PR TITLE
improved compatibility with HIRO-7

### PR DESCRIPTION
**Pull request by sgritsenko**
Adding compatibility with HIRO-7 and the latest version of Decision Service.

### CODE REVIEW

File: python/hiro-client.py [hunk 1]
```diff
@@ -42,16 +43,25 @@ def __init__(self, username, password,
...
     def get(self, url, token=None):
+        url = url.replace('"','') # temporary workaround.
+
         if token is None:
             token, _token_expired = self.get_token()
         headers = self._headers
         if token is not None:
             headers['Authorization'] = "Bearer " + token
         res = requests.get(url, headers=headers, verify=False)
-        return self._parse_response(res)
```
Comment (SteffenBauer) [inline python/hiro-client.py]: Why reverting the handling of the requests response here, from a single method `_parse_response` to individual code blocks for get, post, delete? It looks to me like the handling block ( `if res.status_code == 401 ...return res.text`) is the same for the three access modes get/post/delete, so we have code duplication here.

**Review by SteffenBauer — APPROVED** [review]
Principally okay, just one minor issue with code style (duplicated code which should be moved to a separate method)

File: python/hiro-client.py [hunk 2]
```diff
@@ -42,6 +43,8 @@ def __init__(self, username, password,
 
     @backoff.on_exception(*BACKOFF_ARGS, **BACKOFF_KWARGS)
     def get(self, url, token=None):
+        url = url.replace('"','') # temporary workaround.
```
Comment (mglusiuk) [inline python/hiro-client.py]: could you pls explain where " comes from?

Comment (sgritsenko) [reply python/hiro-client.py]: That's a bit weird, but for example in my IDE if i don't remove ", then it throws an exception from requests library.
At the same time I can successfully run this code directly on the server via console.
This was done mostly with a purpose of compatibility and I plan to improve it the future (re-write how this URL is forming: url = self._graph_endpoint + '/' + quote_plus(node_id))

Screenshot 2020-07-29 at 10 20 20

Comment (mglusiuk) [reply python/hiro-client.py]: maybe you initialize it with quoted graph_endpoint instead of plain string url

**Review by mglusiuk — APPROVED** [review]

**Pull request merged**